packages/wonder-blocks-grid/components/row.md

```diff
@@ -38,3 +38,64 @@ const styles = StyleSheet.create({
     </Row>
 </View>;
 ```
+
+Another example – If the height of the [Cell's](#cell) contents is taller than
+the [Row](#row) it will allow vertical scrolling. 
+
+```jsx
+const Color = require("@khanacademy/wonder-blocks-color").default;
+const {Body} = require("@khanacademy/wonder-blocks-typography");
+const {View, Text} = require("@khanacademy/wonder-blocks-core");
+const {StyleSheet} = require("aphrodite");
+
+const styles = StyleSheet.create({
```

Comment:
I think something more like:
```
const styles = StyleSheet.create({
    view: {
        background: Color.offBlack,
        height: "400px",
    },

    row: {
        padding: "16px 0",
        border: `1px solid ${Color.gold}`,
        background: Color.white,
        height: "100%",
        display: "flex",
        alignItems: "stretch",
    },

    cell: {
        background: Color.gold,
        overflowY: "auto",
        padding: 5,
    },
});
```
would better illustrate that the rows can be styled so that their height fills the parent container.